Rakefile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env rake
 
+module TempFixForRakeLastComment
```

Comment:
2.3.1.  I got it figured out though, if I revert the changes to the lock files and use bundle properly then it will use a compiler that matches our dependencies.  I pushed a commit that backs all of that out.  Thanks for bearing with me while I figure out the build system.